const.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <filesystem>
```

Comment:
I am still not convinced, why are you adding header in const.hpp when the error is in editor_impl.cpp and there is no operation in this file that requires <filesystem>.

Reply:
no reason, checked this PR https://github.com/ibm-openbmc/openpower-vpd-parser/pull/146/files where we add MAX_FILE_SIZE in const and included the missing <filesystem>, as how it is done in the former change.

Reply:
In that case we need to find the appropriate place where this header is required and add it there.

Reply:
done